shell/platform/darwin/macos/framework/Source/FlutterTextInputSemanticsObject.mm

```diff
@@ -124,13 +104,17 @@ - (void)setAccessibilityFocused:(BOOL)isFocused {
 #pragma mark - NSResponder
 
 - (BOOL)becomeFirstResponder {
-  if (_plugin.client == self && [_plugin isFirstResponder]) {
+  FlutterTextInputPlugin* plugin = _engine.viewController.textInputPlugin;
+  if (!plugin) {
+    return NO;
+  }
+  if (plugin.client == self && [plugin isFirstResponder]) {
     // This text field is already the first responder.
     return YES;
   }
   BOOL result = [super becomeFirstResponder];
   if (result) {
-    _plugin.client = self;
+    plugin.client = self;
```

Comment:
I don't see anywhere this in nil'd out; I would expect there to be a flow somewhere that would disconnect this if the engine's FVC is changed (e.g., to nil). Did I miss that, or is it missing?